samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Right now, it is only in the metadata that comes with the principal, so you can get out the Scheme from the authentication description, but its not contained in the actual principal/identities necessarily, some of our middlewares are using authType == authScheme, but that isn't a requirement.